MacOS.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 # Set variables
-DOMAIN="test.com"                   # Active Directory domain name
+DOMAIN="test.com"                   # Domain name
 ADMIN="admin"                       # AD admin username
 PASS="password"                     # AD admin password (not encrypted, use with caution)
```

Comment:
Using a plain text password is a security risk. Consider using a more secure method for handling sensitive information.